modules/ROOT/pages/splitter-flow-control-reference.adoc

```diff
@@ -444,6 +444,14 @@ INFO  YYYY-DD-MM HH:MM:SS,SSS [] org.mule.api.processor.LoggerMessageProcessor:
 INFO  YYYY-DD-MM HH:MM:SS,SSS [] org.mule.api.processor.LoggerMessageProcessor: [[#text: Liam Neeson], [#text: Will Ferrell], [#text: Christian Bale]]
 ----
 
+== Nesting Splitters
+
+Nested splitters override message properties that the Collection Aggregator uses for message assembly.
+
+If you need to split a message multiple times by nesting splitters, store the values from the `correlationId`, `correlationGroupSize`, and `correlationGroupSize` properties before the next splitter in your flow is executed. Then, restore these message properties before the corresponding Collection Aggregator is executed.
```

Comment:
```suggestion
If you need to split a message multiple times by nesting splitters, store the values from the `correlationId`, `correlationGroupSize`, and `correlationGroupSize` properties of the previous splitter flow before the next splitter in your flow is executed. Then, restore these message properties before the corresponding collection aggregator is executed.
```
Should we illustrate this...or have we already?
Do we need to specify to where you store the properties and from where you restore them?